tigerpath/majors_and_certificates/majors/POL.json

```diff
@@ -673,12 +776,53 @@
                     "POL 250",
                     "POL 347",
                     "POL 349",
-                    "POL 352"
+                    "POL 352",
+                    "SPI 329/POL 350"

```

Comment:
https://politics.princeton.edu/track-quantitative-and-analytical-political-science